test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -143,6 +143,11 @@ test
     await t.expect(pageTitle).contains('Verify with your email');
     await t.expect(challengeEmailPageObject.getFormSubtitle())
       .contains('Check your email for a verification message. Click the verification button in your email or enter the code below to continue.');
+
+    // Verify links (switch authenticator link not present since there are no other authenticators available)
+    await t.expect(await challengeEmailPageObject.switchAuthenticatorLinkExists()).notOk();
+    await t.expect(await challengeEmailPageObject.signoutLinkExists()).ok();
+    await t.expect(challengeEmailPageObject.getSignoutLinkText()).eql('Back to sign in');
```

Comment:
There could two types of `Back to sign in` links..  `js-cancel` and `js-goback` . If `cancel` object is not present `js-goback` will be rendered.. just fyi